static/app/views/issueList/draggableTabBar.tsx

```diff
@@ -177,13 +216,16 @@ const makeDefaultMenuOptions = ({onRename, onDuplicate, onDelete}): MenuItemProp
       label: t('Duplicate'),
       onAction: onDuplicate,
     },
-    {
+  ];
+  if (onDelete) {
```

Comment:
Delete option now only shows up if there are more than 2 tabs to prevent a user from deleting all tabs. The backend also enforces this, but we should prevent this on the frontend as well.